src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {

```

Comment:
Ball cannot extend Lottery. Lottery is a service that provides Balls. It's same as Car extends Factory
```suggestion
public class Ball {

```